src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -5259,3 +5259,137 @@ TEST_F(EnsembleFlowCustomNodePipelineExecutionTest, ReloadPipelineWithoutNodeDei
     // in order to count whether deinitialize has been called expected number of times
     ASSERT_EQ(LibraryCountDeinitialize::deinitializeCounter, 3);
 }
+
+
+static constexpr const char* INPUT_TENSOR_NAME = "input_string";
+static constexpr const char* OUTPUT_TENSOR_NAME = "output_string";
+
+struct Passthrough {
```

Comment:
Rename to indicate what is the difference from other mocked nodes
- precision u8
- 2 dimensions

My suggestion is `Passthrough_2D_U8`